compiler-rt/lib/rtsan/rtsan_flags.inc

```diff
@@ -19,3 +19,6 @@
 RTSAN_FLAG(bool, halt_on_error, true, "Exit after first reported error.")
 RTSAN_FLAG(bool, print_stats_on_exit, false, "Print stats on exit.")
 RTSAN_FLAG(const char *, suppressions, "", "Suppressions file name.")
+RTSAN_FLAG(bool, suppress_equal_stacks, true,
```

Comment:
equal => duplicate, maybe?

Reply:
I do agree duplicate is clearer, but this would move away from matching tsan - do we care?

Reply:
OK, let's keep it the same as TSan.